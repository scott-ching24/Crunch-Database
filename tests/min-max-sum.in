min a
max a
sum a
set a 1 2 5 4 3 8 6 7 5
min a
max a
sum a 
get a
bye
