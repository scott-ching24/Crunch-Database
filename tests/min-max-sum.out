> no such key

> no such key

> no such key

> ok

> 1

> 8

> 41

> [1 2 5 4 3 8 6 7 5]

> bye
